rqpy/io/_load.py

```diff
@@ -514,6 +523,53 @@ def get_traces_npz(path):
         
     return traces, info_dict
 
+def load_h5_dump(path, lgcskip_empty=True, lgcreturndict=False):
+    """
+    Function to load HDF5 dumps
+    
+    Parameters
+    ----------
+    path : str
+        Absolute path to dump of traces
+    lgcskip_empty : bool, optional
+        Boolean flag on whether or not to skip empty events. Should be set to false if user only wants the traces.
+        If the user also wants to pull extra timing information (primarily for live time calculations), then set
```

Comment:
I think you have the True/False cases switched around for the lgcskip_empty docstring in load_h5_dump

Reply:
Fixed. I just copied this from `get_traces_midgz`, so I guess it's been wrong for a while...

Reply:
Haha, awesome.